### PR TITLE
Forms Automated Test - LPS-133049 | master

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

File: modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro [hunk 1]
```diff
@@ -675,6 +675,20 @@ definition {
...
+
+		ItemSelector.uploadFile(
+			navTab = "Documents and Media",
+			uploadFileName = "${uploadFileName}");
+
+		AssertElementPresent(
+			key_imageName = "${uploadFileName}",
+			locator1 = "FormViewBuilder#SIDEBAR_ADVANCED_PREDEFINED_IMAGE");
+	}
+
```
Comment (carolmariaabb) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro:691]: Hi @leticiachiu, the App Builder folder has been deleted from Master. Can you move this macro to `FormFields.macro` file and rename it to `setImagePredefinedValue`? Thanks!

File: modules/apps/app-builder/app-builder-test/src/testFunctional/paths/FormViewBuilder.path [hunk 2]
```diff
@@ -233,6 +233,12 @@
 	<td></td>
 </tr>
 
+<tr>
+	<td>SIDEBAR_ADVANCED_PREDEFINED_IMAGE</td>
+	<td>//div[contains(@class,'image-picker-preview')]/img[contains(@src,'${key_imageName}')]</td>
+	<td></td>
+</tr>
+
```
Comment (carolmariaabb) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/paths/FormViewBuilder.path:241]: @leticiachiu, the App Builder folder has been deleted from Master. Can you move this path to another file and rename it to `SIDEBAR_ADVANCED_IMAGE_PREDEFINED_VALUE`? Thanks!

File: modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro [hunk 3]
```diff
@@ -941,6 +955,18 @@ definition {
...
+			key_imageName = "${fileName}",
+			locator1 = "FormFields#IMAGE_PREDEFINED_VALUE");
+
+		if (isSet(fieldDescription)) {
+			AssertElementPresent(
+				key_description = "${fieldDescription}",
+				locator1 = "FormFields#IMAGE_FIELD_DESCRIPTION");
+		}
+	}
+
```
Comment (carolmariaabb) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro:969]: Hi @leticiachiu, the App Builder folder has been deleted from Master. Can you move this macro to FormFields.macro file and rename it to `validateImagePredefinedValue`? Thanks!

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase [hunk 4]
```diff
@@ -131,14 +131,37 @@ definition {
...
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormPortlet.waitFormRenderer(text = "Submit");
+
+		FormViewBuilder.validatePredefinedImage(fileName = "Document_1.jpg");
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase]: @leticiachiu I think the step `Submit the form` is missing.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase [hunk 5]
```diff
@@ -131,14 +131,37 @@ definition {
...
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Image",
+			formPageNumber = "0");
+
+		FormViewBuilder.setFieldPredefinedImage(uploadFileName = "Document_1.jpg");
+
+		Form.save();
+
```
Comment (carolmariaabb) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase:155]: @leticiachiu I think step 5 is missing.. `Assert Document_1.jpg is present in the Image field` just after the form is saved.

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase (hunk 5 above)
Comment (leticiachiu) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase:155]: Hello, I think that I did this step inside of macro setImagePredefinedValue
```
AssertElementPresent(
			key_imageName = "${uploadFileName}",
			locator1 = "Form#SIDEBAR_ADVANCED_PREDEFINED_VALUE");
```

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**